### ISSUE TITLE
Add pricing

### ISSUE THREAD

**Issue by whitingx** [benjystanton/usability-labs#5]
For some of these locations it would be useful to include average pricing to give people an idea of how much hiring the venue might cost prior to direct contact.

**Comment by benjystanton** [comment]
Good idea. I worry if it'll be too much work to maintain though? Happy to include it if you fancy doing he legwork 😁

**Comment by whitingx** [comment]
Sure ツ - for reference _Adam Murray_ has been compiling a similar list in this shared Google Doc;

https://drive.google.com/open?id=1HgaFJyHkQ11jVw7ywjorxAnpTw757GmRylxolDJuQDA

which includes some of the pricing he has obtained. Useful as a reference for this.

**Comment by benjystanton** [comment]
Nice, would Adam be okay for us steal this?
